src/mono/Directory.Build.props

```diff
@@ -24,7 +24,7 @@
 
   <PropertyGroup>
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.1</PackageVersion>
```

Comment:
Should this be here at all? It seems like it should just pull from the eng/Versions.props locations.

Reply:
removed